webviz_subsurface/plugins/_co2_leakage/views/mainview/settings.py

```diff
@@ -0,0 +1,367 @@
+                                ),
+                            ],
+                            style=self.Style.CM_RANGE,
+                        ),
+                        "Maximum",
+                        html.Div(
+                            [
+                                dcc.Input(
+                                    id=cm_max_id,
+                                    type="number"
+                                ),
+                                dcc.Checklist(
+                                    ["Auto"],
+                                    ["Auto"],
+                                    id=cm_max_auto_id,
+                                ),
+                            ],
+                            style=self.Style.CM_RANGE,
+                        ),
+                    ],
+                )
+            ],
+        )
+
+
+class ExperimentalFeaturesLayout(wcc.Selectors):
+    def __init__(self, plume_threshold_id, plume_smoothing_id):
+        super().__init__(
+            label="Experimental",
+            open_details=False,
+            children=[
+                html.Div(
+                    children=[
+                        html.Div("Plume Threshold"),
+                        dcc.Input(
+                            id=plume_threshold_id,
+                            type="number",
+                            min=0,
+                            value=0.000001,
+                            placeholder="Lower property threshold",
+                            style={
+                                "textAlign": "right",
+                            },
+                        ),
+                    ],
+                ),
+                html.Div(
+                    children=[
+                        html.Div("Plume Smoothing"),
+                        dcc.Input(
+                            id=plume_smoothing_id,
+                            type="number",
+                            min=0,
+                            value=0,
+                            step=1,
+                            placeholder="Smoothing [#pixels]",
+                            style={
+                                "textAlign": "right",
+                            },
+                        ),
+                    ],
+                ),
+            ]
+        )
+
+
+class EnsembleSelectorLayout(wcc.Selectors):
+    def __init__(self, ensemble_id, realization_id, ensembles: List[str]):
+        super().__init__(
+            label="Ensemble",
+            open_details=True,
+            children=[
+                "Ensemble",
+                wcc.Dropdown(
+                    id=ensemble_id,
+                    options=[
+                        dict(value=en, label=en)
+                        for en in ensembles
+                    ],
+                    value=ensembles[0],
+                    clearable=False,
+                ),
+                "Realization",
+                wcc.Dropdown(
```

Comment:
For multi selection for many realizations consider using `wcc.SelectWithLabel`

Reply:
Fixed. Agree that this is more user-friendly.